docs/guide/en/toast.md

```diff
@@ -0,0 +1,205 @@
+# Toast
+
+The **Toast** widget renders a [toast](https://getbootstrap.com/docs/5.3/components/toasts/#examples) component.
+
+You can use it to create lightweight notifications similar to push notifications used in mobile and desktop operating
+systems. Toasts are built with flexbox and are easy to align and position.
+
+## Key Features
+- Customizable header with support for images, titles, and timestamps.
+- Simple body content or custom complex content.
+- Optional dismissable functionality with customizable close button.
+- Trigger button integration for displaying toast notifications on demand.
+- Flexible positioning through container utilities.
+
+## Important Note
+Toasts are opt-in for performance reasons, so **you must initialize them with JavaScript** after rendering:
```

Comment:
Can we do something about it?

1. I don't think it is bad for performance.
2. Registering the JS automatically would be great if possible.